e2e/cypress/integration/orbit-blog.spec.js

```diff
@@ -6,12 +6,14 @@ describe('ORBIT Blog', () => {
   beforeEach(() => {
     cy.preserveLoginCookies()
     cy.visit('/')
+    cy.injectAxe()
```

Comment:
thought: I wonder if visit could be made to do this in addition to visiting the url. Though if we switch to playwright this might be moot.